ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/Snapshots.hs

```diff
@@ -217,10 +298,13 @@ deleteSnapshot (SomeHasFS HasFS{..}) = removeFile . snapshotToPath
 -- | List on-disk snapshots, highest number first.
 listSnapshots :: Monad m => SomeHasFS m -> m [DiskSnapshot]
 listSnapshots (SomeHasFS HasFS{..}) =
-    aux <$> listDirectory (mkFsPath [])
+   aux <$> listDirectory (mkFsPath [])
```

Comment:
I think we usually have 4 spaces before a where so I'd keep the space here, not particularly important.